example/App.tsx

```diff
@@ -1,21 +1,28 @@
 import React, {Component} from 'react';
-import {Platform, StyleSheet, Button, ScrollView, View, Image, Text, TouchableOpacity} from 'react-native';
+import {StyleSheet, Button, ScrollView, View, Image, Text, TouchableOpacity} from 'react-native';
 import Purchases from 'react-native-purchases';
 
+type AppState = {
+  currentStatus: string;
+}
 
-export default class App extends Component {
```

Comment:
maybe I should revert the TypeScript changes here, and just clean up a bit the JavaScript example